### ISSUE TITLE
Betting round always show 100 QTUM even 0 qtum betted

### ISSUE THREAD

**Issue by henryzbw** [bodhiproject/bodhi-ui#609]
![image](https://user-images.githubusercontent.com/10339131/41132123-13105b5e-6a74-11e8-93c3-681c6da42e5a.png)

**Comment by dwalintukan** [comment]
it looks like 100 QTUM was bet on the first Oracle/betting round. See the `amounts`?

**Comment by henryzbw** [comment]
@dwalintukan this is wrong number i think. i created a new one to test, still show 100QTUM. nobody betted.

**Comment by henryzbw** [comment]
there is a mismatch between the console.log and http://localhost:5555/graphiql

**Comment by dwalintukan** [comment]
@henryzbw oh gotcha. So the BE might be returning bad data. Let's get this fixed next week's release.
